CORE/world_extensions/core_world.rb

```diff
@@ -60,10 +59,10 @@ def create_new_browser
     def create_chrome_browser
         caps = Selenium::WebDriver::Remote::Capabilities.chrome(chrome_options: {'detach' => true})
 
-        if @configuration['USE_GRID'] == 'true'
-            driver = Selenium::WebDriver.for(:remote, :url => environment['GRID'] , desired_capabilities: caps)
+        if @configuration['USE_GRID']
```

Comment:
My next PR should be pulling all of the browser creation out into it's own location (instead of inside this world object) so maybe that will look a bit cleaner? I am not sure. I agree that it seems like there may be a better way of doing this than the `if @configuration[key]` pattern that seems to be showing up a lot.